builtin/game/item.lua

```diff
@@ -704,7 +704,6 @@ core.nodedef_default = {
 
 	on_receive_fields = nil,
 
-	on_metadata_inventory_move = core.node_metadata_inventory_move_allow_all,
 	on_metadata_inventory_offer = core.node_metadata_inventory_offer_allow_all,
```

Comment:
Why was `move` removed, but not this weird field?

Reply:
Huh, right. But I noticed something else about the other line I removed: this is a commit of yours from 3 years ago: https://github.com/minetest/minetest/blame/025035db5c87e9eaa9f83859f860539fc4fb4dc0/src/script/cpp_api/s_nodemeta.cpp#L157

But 2 years before yours, the callback had been declared obsolete and useless https://github.com/minetest/minetest/blame/025035db5c87e9eaa9f83859f860539fc4fb4dc0/builtin/game/deprecated.lua#L18

So... I'm confused (on_metadata_inventory_move still appears in the API)

Reply:
`move`, `put` and `take` can be removed here. They're superfluous because the C++ code checks against the presence of this field within the node definition. Perhaps the metadata is just ignored.